src/Sustainsys.Saml2/Validation/AssertionValidator.cs

```diff
@@ -84,4 +91,50 @@ protected virtual void ValidateConditions(
         // to care about proxy restrictions. They are only valid if the SP is a proxy that acts as an IDP
         // to other applications. We'll just ignore proxy restrictions for now.
     }
+    /// <summary>
```

Comment:
Insert blank line before the /// for a new method (same in more places in this file)